app/models/manageiq/providers/ansible_tower/automation_manager/configuration_script_source.rb

```diff
@@ -0,0 +1,7 @@
+class ManageIQ::Providers::AnsibleTower::AutomationManager::ConfigurationScriptSource < ConfigurationScriptSource
```

Comment:
to make this work you'll need a `:type` column in `configuration_script_sources`